From 32a9cf86d9f35dc3fb0463199f09087fcb6b8fbf Mon Sep 17 00:00:00 2001
From: Nishanth Menon <nm@ti.com>
Date: Thu, 23 Jul 2015 17:43:37 -0500
Subject: [PATCH 25/25] ARM: dts: am57xx-beagle-x15: Map regulators to voltage
 domains

Map the appropriate SMPS to the correct voltage domains.

Signed-off-by: Nishanth Menon <nm@ti.com>
---
 arch/arm/boot/dts/am57xx-beagle-x15.dts | 22 +++++++++++++++++++++-
 1 file changed, 21 insertions(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/am57xx-beagle-x15.dts b/arch/arm/boot/dts/am57xx-beagle-x15.dts
index 36c0fa6..be35879 100644
--- a/arch/arm/boot/dts/am57xx-beagle-x15.dts
+++ b/arch/arm/boot/dts/am57xx-beagle-x15.dts
@@ -619,8 +619,28 @@
 	ti,no-idle-on-init;
 };
 
+&voltdm_mpu {
+	vdd-supply = <&smps12_reg>;
+};
+
+&voltdm_dspeve {
+	vdd-supply = <&smps45_reg>;
+};
+
+&voltdm_gpu {
+	vdd-supply = <&smps45_reg>;
+};
+
+&voltdm_ivahd {
+	vdd-supply = <&smps45_reg>;
+};
+
+&voltdm_core {
+	vdd-supply = <&smps6_reg>;
+};
+
 &cpu0 {
-	cpu0-supply = <&smps12_reg>;
+	cpu0-voltdm = <&voltdm_mpu>;
 	voltage-tolerance = <1>;
 };
 
-- 
2.7.0.rc3

